src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {

```

Comment:
Override annotation is optional but it's better to use it to explicitly pay attention to fact that methods are inherited from parent class or implemented interface